cmd/podman/create.go

```diff
@@ -547,7 +547,10 @@ func parseCreateOpts(c *cli.Context, runtime *libpod.Runtime, imageName string,
 	if len(entrypoint) == 0 {
 		entrypoint = data.ContainerConfig.Entrypoint
 	}
-
+	// if entrypoint=, we need to clear the entrypoint
```

Comment:
nit comment has 'entrypoint=,'  Should it be "If the entrypoint has no value" or perhaps entrypoint is nil?

Reply:
only if you want me to try and pass all the tests again :)

Reply:
Touch up in some unrelated future PR is fine by me, I trust ya.